### PR TITLE
Tidying up comments across all scripts

### PR DESCRIPTION
**Pull request by stokesriona**

### CODE REVIEW

File: maintain.js [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
-        // console.log(lines)
-        //-------------------------------------------
-        
         const checkOps = checkOperations()
 
         async function runChecks() {
             let results = {}
             
-            // reading client's files in
+            // reading client's JSON files in
```
Comment (rjrodger) [inline maintain.js:21]: we only need to worry about the top level package.json file

File: maintain.js [hunk 2]
```diff
@@ -208,10 +175,8 @@ class Maintain {
                                 chain.push(searchContent[i])
                             }
                             pass = (null != (Hoek.reach(fileContent,chain)))
-                            // console.log(pass)
         
-                            // add in else if clause for if searching for json value
-                        } else {
+                        } else { // add in "else if" clause if searching for json value
```
Comment (rjrodger) [inline maintain.js:179]: prefer the style:
```
if() {
}
else if() {
}
else {
}
```

makes it easier to cut and paste if-else clauses

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: maintain.js (hunk 1 above)
Comment (stokesriona) [reply maintain.js:21]: That's changed now to only take the top level .json files, depending on future checks I may refactor how it's collecting the files in the first place

File: maintain.js (hunk 2 above)
Comment (stokesriona) [reply maintain.js:179]: They're all split up now for this script, I'll do the same for the rest of the files in another cleanup branch

**Pull request merged**